### PR TITLE
Add Symfony 4 support

### PR DESCRIPTION
**Pull request by surikman**
This PR adds support for the newest version of Symfony 4.0.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +31,7 @@ env:
 
 before_script:
     - composer self-update
+    - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
```
Comment (keradus) [inline .travis.yml]: no need to install whole symfony full stack when we actually rely on 3 components only

**Comment by surikman** [comment]
finally passed

**Comment by emirb** [comment]
Thanks @surikman.

**Comment by debo** [comment]
When will this be tagged for release?

**Pull request merged**